docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,681 @@
+* Good, because it works similar to zone-originated policies (we implicitly add `kuma.io/zone` tag for top-level targetRef)
+* Bad, because it doesn't remove necessity to create policies in `kuma-system` namespace
+
+## Pros and Cons of adopting producer/consumer concept from GAMMA
+
+* Good, because it's familiar to users who know GAMMA
+* Good, because creating policies in `kuma-system` becomes an edge use-case, the vast majority of use cases is covered by namespace-scoped policies
+* Bad, because it requires syncing policies across zones. 
+  Today zone-originated policies are synced only to global for visibility. 
+  It will be possible to apply policy to `zone-a` and it'll be synced to `zone-b`.
+
+## Implementation
+
+### Producers and consumers
+
+[GAMMA](https://gateway-api.sigs.k8s.io/mesh/#gateway-api-for-service-mesh) introduces a concept of producer and consumer routes:
+
+> A Route in the same Namespace as its Service is called a **producer route**, 
+> since it is typically created by the creator of the workload in order to define acceptable usage of the workload.
+
+> A Route in a different Namespace than its Service is called a **consumer route**. 
+> Typically, this is a Route meant to refine how a consumer of a given workload makes request of that workload.
+
+The concept of producer/consumer routes is very convenient. 
+We can extend this to producer/consumer policies in general:
+
+* A **producer policy** is a policy created in the same namespace as the service it's referring to. 
+Typically created by the service owner in order to define acceptable usage of the service.
+
+* A **consumer policy** is a policy created in a different namespace than the service it's referring to.
+Typically, the policy is meant to refine how a consumer of a given service makes request to it.
+
+Applying this concept to Kuma policies, we'll get the following examples:
+
+```yaml
+# Backend Service Owner creates a producer policy 
+metadata:
+  namespace: backend-ns
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+        namespace: backend-ns
+      default: $conf1 # 'conf1' is going to be applied to all consumers of the backend service
+---
+# Frontend Service Owner creates a consumer policy 
+metadata:
+  namespace: frontend-ns
+spec:
+  targetRef:
+    kind: MeshSubset
+    tags:
+      kuma.io/service: frontend_frontend-ns_svc_8080
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+        namespace: backend-ns
+      default: $conf2 # 'conf2' refines 'conf1' that was already provided by the producer
+```
+
+We can formally define what's producer and consumer Kuma policies:
+
+* if for each `idx` the value of `spec.to[idx].targetRef.namespace` is equal to `metadata.namespace` then it's a producer policy
+* for each `idx` the value of `spec.to[idx].targetRef.namespace` is **not** equal to `metadata.namespace` then it's a consumer policy
```

Comment:
Is this 
```suggestion
* if for some `idx` the value of `spec.to[idx].targetRef.namespace` is **not** equal to `metadata.namespace` then it's a consumer policy
```

i.e. the negation of the previous line

Reply:
It shouldn't be possible to mix producer and consumer to-items. That's why the second is not a negation of the first line